src/api/protectedApiClient.ts

```diff
@@ -33,6 +38,39 @@ export interface ProtectedApiClient {
     newLevel: string;
     password: string;
   }) => Promise<void>;
+  readonly createTeam: (request: {
+    name: string;
+    bio: string;
+    inviteEmails: string[];
+  }) => Promise<void>;
```

Comment:
Is there an interface defined for this request object? If so we should reference it

Reply:
same thing for other cases